main.go

```diff
@@ -13,12 +13,13 @@ import (
 
 	"github.com/docker/docker/pkg/term"
 	"github.com/shibbybird/eazy-ci/lib/builders"
+	"github.com/shibbybird/eazy-ci/lib/runtimes"
 	"github.com/shibbybird/eazy-ci/lib/utils"
 
 	"github.com/shibbybird/eazy-ci/lib/config"
 )
 
-var VERSION = "v0.0.2"
+var version = "v0.0.2"
```

Comment:
this was giving me a lint error and annoyed me. :)

Reply:
Yeah... I'm too much of a polyglot at this point, and can't remember/don't know all of the best practices.

Reply:
Me either, I actually though this was the go standard for consts (cause C) until the linter freaked out.